src/Tasks/PublicAPI/net/PublicAPI.Shipped.txt

```diff
@@ -2242,6 +2242,8 @@ Microsoft.Build.Tasks.XslTransformation.XslCompiledDllPath.get -> Microsoft.Buil
 Microsoft.Build.Tasks.XslTransformation.XslCompiledDllPath.set -> void
 Microsoft.Build.Tasks.XslTransformation.XslContent.get -> string
 Microsoft.Build.Tasks.XslTransformation.XslContent.set -> void
+Microsoft.Build.Tasks.XslTransformation.PreserveWhitespace.get -> bool
+Microsoft.Build.Tasks.XslTransformation.PreserveWhitespace.set -> void
```

Comment:
These new lines should go into `PublicAPI.Unshipped.txt`, since they haven't (yet!) gone into a public release. The distinction helps us understand what public API we can modify (things that haven't yet released), and what we can't (changing or removing things that have been released would be a breaking change).

If you use Visual Studio or another IDE that can handle Roslyn code fixers, this should be available as a "quick fix" action.

Reply:
I'm using vscode, so I edited them manually. I moved them from `PublicAPI.Shipped.txt` to `PublicAPI.Unshipped.txt`